wazo_acceptance/assets/2_users.json

```diff
@@ -0,0 +1,130 @@
+    "supervision_enabled": true,
+    "call_transfer_enabled": false,
+    "dtmf_hangup_enabled": true,
+    "call_record_outgoing_external_enabled": true,
+    "call_record_outgoing_internal_enabled": true,
+    "call_record_incoming_internal_enabled": true,
+    "call_record_incoming_external_enabled": true,
+    "online_call_record_enabled": false,
+    "simultaneous_calls": 5,
+    "ring_seconds": 30,
+    "userfield": "userfield",
+    "call_permission_password": "1234",
+    "enabled": true,
+    "username": "richardlapointe",
+    "password": "secret",
+    "auth": {
+      "username": "richardlapointe",
+      "password": "secret"
+    },
+    "lines": [
+      {
+        "context": "default",
+        "position": 2,
+        "provisioning_code": "887865",
+        "extensions": [
+          {
+            "context": "default",
+            "exten": "1100"
+          }
+        ],
+        "endpoint_sip": {
+          "extra": "content_generated_by_code"
+        }
+      }
+    ],
+    "incalls": [
+      {
+        "extensions": [
+          {
+            "exten": "4100",
+            "context": "from-extern"
+          }
+        ]
+      }
+    ],
+    "groups": [
+      {
+        "uuid": "content_generated_by_code"
+      }
+    ],
+    "switchboards": [
+      {
+        "uuid": "content_generated_by_code"
+      }
+    ]
+  },
+  {
+    "subscription_type": 2,
+    "firstname": "Celine",
+    "lastname": "Dion",
+    "email": "celinedion@wazo.io",
+    "language": "fr_FR",
+    "outgoing_caller_id": "\"Celine Dion\" <4185551234>",
+    "mobile_phone_number": "4181234569",
+    "supervision_enabled": true,
+    "call_transfer_enabled": false,
+    "dtmf_hangup_enabled": true,
+    "call_record_outgoing_external_enabled": true,
+    "call_record_outgoing_internal_enabled": true,
+    "call_record_incoming_internal_enabled": true,
+    "call_record_incoming_external_enabled": true,
+    "online_call_record_enabled": false,
+    "simultaneous_calls": 5,
+    "ring_seconds": 30,
+    "userfield": "userfield",
+    "call_permission_password": "1234",
+    "enabled": true,
+    "username": "celinedion",
+    "password": "secret",
+    "auth": {
+      "username": "celinedion",
+      "password": "secret"
+    },
+    "lines": [
+      {
+        "context": "default",
+        "position": 2,
+        "provisioning_code": "887866",
+        "extensions": [
+          {
+            "context": "default",
+            "exten": "1101"
+          }
+        ],
+        "endpoint_sip": {
+          "extra": "content_generated_by_code"
+        }
+      }
+    ],
+    "incalls": [
+      {
+        "extensions": [
+          {
+            "exten": "4101",
+            "context": "from-extern"
+          }
+        ]
+      }
+    ],
+    "groups": [
+      {
+        "uuid": "content_generated_by_code"
+      }
+    ],
+    "switchboards": [
+      {
+        "uuid": "content_generated_by_code"
+      }
+    ]
+  }
+]
```

Comment:
By default all our file have newline at end of line
You can configure your editor